engine/src/test/java/io/camunda/zeebe/engine/util/StreamPlatform.java

```diff
@@ -52,6 +53,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
+import org.mockito.Mockito;
 import org.slf4j.Logger;
 
 public final class StreamPlatform {
```

Comment:
The changes in this class where necessary in order to make our test work. They are already available on main.